<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>占道停车服务器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>931</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPort">
     <attribute name="title">
      <string>串口信息</string>
     </attribute>
     <widget class="QPushButton" name="btnPortCmd">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>12</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>发送命令</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>12</y>
        <width>374</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblCmd0">
         <property name="text">
          <string>检测器信道号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbCmd0">
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCmd1">
         <property name="text">
          <string>检测器信道号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbCmd1">
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCmd2">
         <property name="text">
          <string>检测器信道号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbCmd2">
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>12</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>M77R/M77指令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbM77Cmd"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbComM77R"/>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>901</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>静态信息</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txtPortStaticLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>371</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>动态信息</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txtPortDynamicLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabNetwork">
     <attribute name="title">
      <string>网络信息</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>11</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btnStartListen">
         <property name="text">
          <string>启动侦听</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopListen">
         <property name="text">
          <string>停止侦听</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>侦听端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbPort">
         <property name="minimum">
          <number>1024</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>6000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>侦听等待最大连接数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbMaxConn">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>901</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>静态信息</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txtNetworkStaticLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>371</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>动态信息</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txtNetworkDynamicLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDatabase">
     <attribute name="title">
      <string>数据库信息</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>11</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="btnHide">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnConnectDb">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>341</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据库服务器IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtServerIP">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>服务端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbServicePort">
         <property name="minimum">
          <number>1024</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>3306</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>901</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>静态信息</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txtDatabaseStaticLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>371</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>动态信息</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txtDatabaseDynamicLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
